config/docker/build-base/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 
 # Python 3.7
 RUN apt-get update && apt-get install --no-install-recommends -y \
-    python3.7 \
+    python3.9 \
```

Comment:
Please extend the GitHub checks to include Python 3.9 as at the moment it's only doing 3.6, 3.7 and 3.8:
https://github.com/kernelci/kernelci-core/blob/main/.github/workflows/main.yml#L22

Maybe we can also drop 3.6 now.

Reply:
done

Reply:
Thanks.  It would have been better as a separate commit, and also with prefixes in the commit messages (e.g. config/docker for the Dockerfile changes, and github/workflows for the checks).